src/renderer/views/app/AppView.tsx

```diff
@@ -73,9 +73,9 @@ export const AppView: React.FC = (): JSX.Element => {
             }
           }) => {
             let msg = ''
-            msg = haltEthTrading ? 'Trading for ETH is halted temporarily.' : msg
-            msg = haltTrading ? 'Trading for all pools is halted temporarily.' : msg
-            msg = haltThorChain ? 'THORChain is halted temporarily.' : msg
+            msg = haltEthTrading ? intl.formatMessage({ id: 'halt.trading.eth' }) : msg
```

Comment:
[nitpick] We could use `halt.chain` here as well (in favour of removing `halt.trading.eth`).